lamindb_setup/dev/upath.py

```diff
@@ -397,10 +399,11 @@ def convert_pathlike(pathlike: Union[str, Path, UPath]) -> UPath:
 def create_path(path: UPath) -> UPath:
     global AWS_CREDENTIALS_PRESENT
 
+    print(f"create path: {path}")
     path = convert_pathlike(path)
     if not isinstance(path, S3Path):
         return path
-    path = UPath(path, cache_regions=True)
+    path = UPath(path)
```

Comment:
I guess this just here because we wanted to test without it. Should be restored i believe.

Reply:
Yes, I just restored it.